### PR TITLE
added analytics

### PR DESCRIPTION
**Pull request by Greatseun02**
@Tiny999

### CODE REVIEW

File: src/checkEnv.js
Comment (Tiny999) [inline src/checkEnv.js:1]: what's this file for @Greatseun02 
Please remove

File: src/utils/analytics.js
Comment (Tiny999) [inline src/utils/analytics.js:1]: @Greatseun02 Why do you have so much unnenecessary code here? 

Why does it have to be so complex? Can you walk me through this file

File: src/utils/analytics.ts
Comment (Tiny999) [inline src/utils/analytics.ts:1]: Why do we have an analytics.ts and an analytics.js file in the same project.

Also, why do you have a typescript file in a javascript project. Did this compile for you??

**Review by Tiny999 — CHANGES_REQUESTED** [review]

File: src/checkEnv.js
Comment (Greatseun02) [reply src/checkEnv.js:1]: Sorry about that, I wanted to try out something.

File: src/utils/analytics.js
Comment (Greatseun02) [reply src/utils/analytics.js:1]: That is just the compiled file of the typescript file.

File: src/utils/analytics.ts
Comment (Greatseun02) [reply src/utils/analytics.ts:1]: There was a conflict, I copied the code Omo sent to me which was in typescript so I created a typescript file to accommodate the code.

**Comment by Greatseun02** [comment]
I have removed the checkEnv.js file from the repo. I have also turned the typescript file to a javascript file. I made some minor change to the code in setUserParams which you can check out. @Tiny999

**Review by Tiny999 — APPROVED** [review]

**Pull request merged**